packages/atlas/src/components/_crt/RevenueShareParticipationWidget.tsx

```diff
@@ -58,22 +73,25 @@ export const RevenueShareParticipationWidget = () => {
                 ENDED ON
               </Text>
               <Text variant="h400" as="h4">
-                {holdersUnlocked}/{holders}
+                {revenueShare.stakers.length}/69420
```

Comment:
should we get rid of the hardcoded value here(and below in the file)?

Reply:
I'd leave it as it is and fix everything when we will have access to the transferable balances etc